include/revng/ADT/SortedVector.h

```diff
@@ -326,9 +326,10 @@ class SortedVector {
     }
 
   protected:
-    T &insertImpl(const T &Value) {
+    template<typename... Types>
+    T &emplaceImpl(Types &&...Values) {
```

Comment:
What do you think?

[SortedVector: improve batch inserters](https://github.com/revng/revng/pull/232/commits/e798e4ddef0c3741bb5006d84f0bf784eef9d98c)
```
Now using them allows populating the underlying vector by moving
values in or constructing them in-place instead of only being able
to copy them in.
```